Source/Src_nd/compute_dt.F90

```diff
@@ -472,14 +475,14 @@ subroutine firstdt(lev, dt, umax, lo, hi, dx, &
        else
           dt_sound = cfl / max(spdx,spdy,spdz)
        end if
-       call amrex_min(dt,dt_sound)
+       dt = min(dt,dt_sound)
     end if
```

Comment:
don't we need `amrex_min` still?

Reply:
I remember when I was writing this there were some changes being made to the dt routines, so I probably ended up merging the wrong thing at some point. There shouldn't be any changes in the dt stuff compared to development, so I'll fix this